railties/test/application/rake_test.rb

```diff
@@ -129,6 +129,8 @@ def test_rake_routes_calls_the_route_inspector
         end
       RUBY
 
+      add_to_config "config.action_cable.routes_enabled = false"
```

Comment:
I think we need to do the same thing Sprockets does to mark the route as internal